public/components/trace_analytics/components/traces/trace_view.tsx

```diff
@@ -38,6 +38,9 @@ import { PanelTitle, filtersToDsl, processTimeStamp } from '../common/helper_fun
 import { ServiceMap, ServiceObject } from '../common/plots/service_map';
 import { ServiceBreakdownPanel } from './service_breakdown_panel';
 import { SpanDetailPanel } from './span_detail_panel';
+import { coreRefs } from '../../../../framework/core_refs';
+
+const newNavigation = coreRefs.chrome?.navGroup.getNavGroupEnabled();
```

Comment:
this is outside of the react lifecycle, can it get correct value of whether nav group is enabled?

Reply:
It does, have tested turning it on and off and the page renders correctly when returning to it.